### ISSUE TITLE
Training set Images

### ISSUE THREAD

**Issue by mayuri2307** [sairajk/PyTorch-Pyramid-Feature-Attention-Network-for-Saliency-Detection#10]
Hi,
Thanks for the wonderful code. I wanted to ask that is there any way to create the training set images? Like how can we create salient images that can be fed into our training model?

**Comment by sairajk** [comment]
I believe you should find tools exactly for this or a similar purpose on GitHub or Google.

For instance, [Mathworks Semantic Segmentation Labeler](https://www.mathworks.com/help/vision/ug/label-pixels-for-semantic-segmentation.html).
